packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -210,11 +210,6 @@ class AnalyzeCommand extends FlutterCommand {
       return false;
     }
 
-    // Don't run pub if asking for android analysis.
```

Comment:
Can you add some more context to the description why this command does a partial android build?

Reply:
added a more straight forward if case